src/js/newsletter/components/ConditionsCheckbox.js

```diff
@@ -0,0 +1,30 @@
+import React from "react"
+
+export default function ConditionsCheckbox(props) {
+  return (
+    <div className="control">
+      <input
+        {...props}
+        type="checkbox"
+        id="conditions"
+        className={`checkbox is-big-checkbox ${props.className}`}
+      />
+      <label htmlFor="conditions" className="has-text-justified is-size-7">
+        By checking this box you acknowledge that your information will be
+        transferred to Mailchimp for processing and agree to our&nbsp;
+        <a href="/terms-conditions" target="_blank">
```

Comment:
I think this should not open in a new tab, but maybe that is my personal opinion.

Reply:
Yes this was a topic in a meeting and Sash said that for the user experience such links should open in new tabs. That's why it is like that. :man_shrugging: